runtime/autoload/man.vim

```diff
@@ -64,8 +64,9 @@ function! man#open_page(count, count1, mods, ...) abort
     return
   endtry
 
-  call s:push_tag()
-  let bufname = 'man://'.name.(empty(sect)?'':'('.sect.')')
+  let fullname = name.(empty(sect)?'':'('.sect.')')
+  call s:push_tag(fullname)
```

Comment:
I suppose this is existing behaviour, but you could argue that we don't want to push a tag or touch the tag stack at all now - `'tagfunc'` handles this all for us.

My reasoning is that as a user, would I expect the tag stack to change when I run a `:Man` command? Particularly as we now have `:tag <man page>` available, at least in man-page buffers. I'd be interested to hear your thoughts.

Reply:
Either way is ok. Maybe keep it as is.

Reply:
I haven't actually tried it but it should be possible to do

```vim
    let l:page = 'vim'
    let [l:buf, l:save_tfu] = [bufnr(), &tagfunc]
    try
        set tagfunc=ManTagFunc
        execute "tag" l:page 
    finally
        call setbufvar(l:buf, '&tagfunc', l:save_tfu))
    endtry
```

Reply:
Great - I've gone with that. Everything should be ready now, pending another review round.